### ISSUE TITLE
biomal model dataset split

### ISSUE THREAD

**Issue by wawpaopao** [ejarmand/comparative_epigenomic_motor_cortex#3]
‘’‘’To evaluate the ability of the model to predict epigenomes in unseen species, we trained four bimodal models, one excluding each species (for example, to evaluate unseen prediction accuracy in mouse, we trained a model using human, macaque and marmoset data). We then evaluated the model using the test dataset of the unseen species. ‘’‘’
Here, if the unseen species is human, does this mean that the model is trained using the data from the other three species and then tested on all peaks in human, or only on the test peaks in human (chromosomes 14, 4, X)? Also, during training, is the model trained using all peaks from the three species, or only using the training dataset from the three species?

**Comment by ejarmand** [comment]
I wawpaopao, It's trained only using the training set for all species, although checkpoints are identified from the best validation loss. After training, the model is evaluated only on the human test set (chromosomes 14, 4, x) to minimize accuracy inflation from leaked regions.
